### PR TITLE
[WIP] Add logic to build mutiarch origin-release

### PR DESCRIPTION
**Pull request by jeffdyoung**
Adding logic so users can compile centos7 origin-release locally for ppc64le, aarch64 and (someday) s390x.

### CODE REVIEW

File: projects/origin-release/golang-1.10/Dockerfile [hunk 1]
```diff
@@ -26,9 +26,19 @@ RUN yum install -y epel-release && \
 
 ADD *.rpm /tmp/origin-rpm/
 RUN yum install -y /tmp/origin-rpm/*.rpm && \
-    yum clean all && \
-    curl https://storage.googleapis.com/golang/go$VERSION.linux-amd64.tar.gz | tar -C /usr/local -xzf - && \
-    go get golang.org/x/tools/cmd/cover \
+    yum clean all 
+
+#Download go for amd64 unless other archs are detected
```
Comment (stevekuznetsov) [inline projects/origin-release/golang-1.10/Dockerfile]: Would we rather see `Dockerfile.ppc64le` than `case` statements on every point of variation between image manifests?

Comment (smarterclayton) [reply projects/origin-release/golang-1.10/Dockerfile]: Yes

Comment (smarterclayton) [reply projects/origin-release/golang-1.10/Dockerfile]: Each architecture should have its own docker file

Comment (jeffdyoung) [reply projects/origin-release/golang-1.10/Dockerfile]: This implies that (in the future) amd64 should have its own Dockerfile. 
Long term, I think its better to maintain a Dockerfile per OS base (rhel/centos and soon fedora). If there are arch specific dependencies, they should be addressed at the repo level first, and if that isn't possible (like this case) then logically either in the dockerfile or as a parameter passed at build time.
FWIW, the origin, web-console-server, and image-registry Dockerfile's work as is, so I don't expect this logic to be in a lot of places.

I think the larger issue here is that there hasn't been a roadmap discussed for how these 'other' archs are to be integrated, and manifest listed.

**Comment by jeffdyoung** [comment]
Reverted initial changes. We've decide to move the multi-arch work to fedora based containers.

**Comment by stevekuznetsov** [comment]
@smarterclayton

**Comment by jeffdyoung** [comment]
@smarterclayton bump :)
Will this container be ubi based soon?

**Comment by droslean** [comment]
@smarterclayton can you check this one? This PR is hanging for many months now.

**Comment by jeffdyoung** [comment]
Leaving redhat, so I'm going to close this. If you want to build OKD on other archs you'll have to manually build this release container first. You can use this as reference:
https://github.com/multi-arch/openshift-fedora/blob/master/groups/origin-release/golang-1.10/Dockerfile.f29

cc: @andymcc @dbenoit17 @ausil

**Pull request closed without merging**